api_EPA.py

```diff
@@ -57,6 +74,25 @@ def main():
 
     release_dataset()
 
+def get_start_time():
+
+    client = bigquery.Client()
+
+    start_time = """SELECT DATETIME_ADD(MAX(PARSE_DATETIME('%Y-%m-%dT%H:%M', UTC)), INTERVAL 1 hour) as next_dt 
+                    FROM redivis.epa_test.pm2_5:1"""
+
+    query_job = client.query(start_time)
+
+    print("Fetching EPA data")
+    for row in query_job:
+        print("EPA Data Retrieved")
```

Comment:
Ditto. Also, I'm not particularly familiar with python iterators, but is there a better way to get the first row from the query_job results?